api/services/SurveyResultService.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const Service = require('trails-service');
+
+/**
+ * @module ResultService
+ * @description candidate match results
+ */
+module.exports = class SurveyResultService extends Service {
+    get() {
+        return this.app.orm.SurveyResult.findAll();
+    }
+
+    match(params) {
+        return this.app.orm.SurveyResult.find({
+            where: params
+        })
+        .then((result) => {
+            const query = 'select surveyresult.id as "SurveyResultId"' +
```

Comment:
we can leverage using ES6 strings to use multiline template 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals